### PR TITLE
[FIX] web: make calendar toggle work

### PR DESCRIPTION
**Pull request by hdh-odoo**
**Description of the issue/feature this PR addresses:**
- Task: https://www.odoo.com/web#id=1975004&action=333&active_id=1278&model=project.task&view_type=form&menu_id=4720
- Pad: https://pad.odoo.com/p/r.916220eeaa9db66e51ebf54cbf7bc24a

**Current behavior before PR:** Mini calendar won't disappear when you click on the cross sign.

**Desired behavior after PR is merged:** Mini calendar would disappear when you click on the cross sign.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by sswapnesh** [comment]
Fixes https://github.com/odoo/odoo/issues/27266

File: addons/web/static/tests/views/calendar_tests.js [hunk 1]
```diff
@@ -2604,6 +2606,39 @@ QUnit.module('Views', {
...
+            "sidebar should be opened");
+
+        // click on close
+        testUtils.dom.click(calendar.$('.o_calendar_sidebar_toggler'));
+        fullWidth = calendar.call('local_storage', 'getItem', 'calendar_fullWidth');
+        assert.strictEqual(fullWidth, true,
+            "calendar_fullWidth should be true in local storage");
+        assert.containsOnce(calendar, '.o_calendar_sidebar_container.o_sidebar_hidden',
+            "sidebar should be hidden");
+
```
Comment (aab-odoo) [inline addons/web/static/tests/views/calendar_tests.js]: maybe trigger a reload and check that the sidebar is still folded

Comment (msh-odoo) [reply addons/web/static/tests/views/calendar_tests.js]: Done 👍

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**